core/modules/user/views/user.views.inc

```diff
@@ -126,15 +126,15 @@ function user_views_data() {
     'filter' => array(
       'handler' => 'views_handler_filter_string',
       'title' => t('Name (raw)'),
-      'help' => t('The user or author name. This filter does not check if the user exists and allows partial matching. Does not utilize autocomplete.')
+      'help' => t('The user account or author name. This filter does not check if the user account exists and allows partial matching. Does not utilize autocomplete.')
```

Comment:
Whatever we do above, this should match. It reads okay to me here, maybe we leave it as-is?  Up to you!